scala/private/phases/phases.bzl

```diff
@@ -0,0 +1,116 @@
+load(
```

Comment:
It's mostly just for convenience. Regarding the underscore, I'm pretty sure you have to do it this way (or similar) to appease modern Bazel.

Reply:
Re exporting you're right, I thought they kept a way to say export is intentional but they haven't https://github.com/bazelbuild/bazel/issues/5636
Re convenience- the downside of it is coupling since this file depends on basically everything. Not sure here...

Reply:
It's mostly for convenience +1.

Currently if we do 
```
load(
    "@io_bazel_rules_scala//scala/private:phases/phases.bzl",
    "extras_phases",
    "phase_binary_compile",
    "phase_binary_final",
)
```
without it we have to do
```
load(
    "@io_bazel_rules_scala//scala/private:phases/api.bzl",
    "extras_phases",
)
load(
    "@io_bazel_rules_scala//scala/private:phases/phase_compile.bzl",
    "phase_binary_compile",
)
load(
    "@io_bazel_rules_scala//scala/private:phases/phase_final.bzl",
    "phase_binary_final",
)
```